content/_index.html

```diff
@@ -9,117 +9,102 @@
 ---
 
 <h2 class="mbs">Explore our APIs</h2>
-<div class="flex flex-wrap mbxl">
-  <a
-    href='{{< relref "api/shipping-guide_2" >}}'
-    class="mb-card flex flex-dir-col flex-1-1-20r pam mrxs mbxs"
-  >
+<div class="flex flex-wrap gaxs mbxl">
+  <div class="bg-green1 flex flex-dir-col start__col pam">
     <div class="flex align-ic mlxs">
       <span
         data-mybicon="mybicon-list"
         data-mybicon-class="icon-ui--green-dark mrs"
         data-mybicon-width="19"
         data-mybicon-height="19"
       ></span>
-      <h3 class="mlxs fwn">Shipping Guide API</h3>
+      <h3 class="mlxs fwn">Checkout</h3>
     </div>
     <p class="mhxs pll">
       Look up available services, delivery times, prices and environmental data,
       and use our recommended service descriptions in your checkout.
     </p>
-  </a>
+  </div>
 
-  <a
-    href='{{< relref "api/booking" >}}'
-    class="mb-card flex flex-dir-col flex-1-1-20r pam mrxs mbxs"
-  >
+  <div class="bg-green1 flex flex-dir-col start__col pam">
     <div class="flex align-ic mlxs">
       <span
         data-mybicon="mybicon-box"
         data-mybicon-class="icon-ui--green-dark mrs"
         data-mybicon-width="19"
         data-mybicon-height="19"
       ></span>
-      <h3 class="mlxs fwn">Booking API</h3>
+      <h3 class="mlxs fwn">Booking</h3>
     </div>
     <p class="mhxs pll">
       Create shipments and generate shipment number, tracking link, EDI-prenotification, label and other transport documents. Order pickups.
     </p>
-  </a>
+  </div>
 
-  <a
-    href='{{< relref "api/pickup-point" >}}'
-    class="mb-card flex flex-dir-col flex-1-1-20r pam mrxs mbxs"
-  >
+  <div class="bg-green1 flex flex-dir-col start__col pam">
     <div class="flex align-ic mlxs">
       <span
-        data-mybicon="mybicon-location"
+        data-mybicon="mybicon-search"
         data-mybicon-class="icon-ui--green-dark mrs"
         data-mybicon-width="19"
         data-mybicon-height="19"
       ></span>
-      <h3 class="mlxs fwn">Pickup Point API</h3>
+      <h3 class="mlxs fwn">Tracking</h3>
     </div>
     <p class="mhxs pll">
-      Let your customers choose their preferred pickup point in your checkout.
+      Look up tracking events of your shipments, or subscribe to them, and make them available to your customers.
```

Comment:
👍 just a slight addition about webhooks to signify the difference between looking up and getting updates pushed.

```suggestion
      Look up tracking events of your shipments, or subscribe to them, and make them proactively available to your customers through webhooks.
```